src/local/experiment.py

```diff
@@ -203,9 +203,9 @@ def eval_and_log(self, metric_fun=r2_score, metric_name='r2'):
         val_preds = torch.cat(val_preds).squeeze().cpu().numpy()
         test_preds = torch.cat(test_preds).squeeze().cpu().numpy()
                 
-        metric_train = metric_fun(y_true=self.y.train, y_pred=train_preds)
-        metric_val = metric_fun(y_true=self.y.val, y_pred=val_preds)
-        metric_test = metric_fun(y_true=self.y.test, y_pred=test_preds)
+        metric_train = metric_fun(self.y.train, train_preds)
```

Comment:
@dkolobok `roc_auc_score`'s second argument is `y_score`, not `y_pred`, so I removed the keyword arguments here. I don't like it, but I thought it'd be the lesser evil that lets us keep the same interface.